packages/scss/src/components/_button.scss

```diff
@@ -123,6 +123,19 @@
 		}
 	}
 
+	&.mod-more {
+		&::before {
+			content: "";
+			@include makeIcon("arrow_south");
+			position: absolute;
+			top: 0;
+			left: 0;
+			font-size: .7rem;
```

Comment:
does this work with the different sizes of buttons

Reply:
And you are setting top/left twice (0 then .75.rem)